www/js/lib/zimfile.js

```diff
@@ -81,16 +81,36 @@ define(['xzdec_wrapper', 'util', 'utf8'], function(xz, util, utf8) {
      */
     ZIMFile.prototype._readSlice = function(offset, size)
     {
-        var fileNumber = 0;
-        var offsetInActualFile = offset;
-        var fileSize = this._files[0].size;
-        if (this._files.length > 1 && offset > fileSize) {
-            // We need to find in which actual file the data must be read
-            fileNumber = Math.floor(offset / fileSize);
-            offsetInActualFile = offset - fileNumber * fileSize;
```

Comment:
That's weird. Are you saying the regression also exists for single-file archives?